docs/markdown/Getting Help/the-pants-community/team.md

```diff
@@ -4,6 +4,7 @@ slug: "team"
 excerpt: ""
 hidden: false
 createdAt: "2022-08-15T22:36:48.659Z"
+updatedAt: "2022-09-08T00:28:12.217Z"
```

Comment:
I think we could leave this field out, and have readme add it at the time of sync instead.

Reply:
Yeah, that was a mistake. I've updated it. But are you saying we have the option to leave it out entirely? 'Cuz I like that even better.

Reply:
Woot! Deleted.